src/sinol_make/helpers/package_util.py

```diff
@@ -143,3 +152,30 @@ def get_memory_limit(test_path, config, lang, args=None):
 
     str_config = util.stringify_keys(config)
     return _get_limit(LimitTypes.MEMORY_LIMIT, test_path, str_config, lang)
+
+
+def validate_files(task_id):
+    """
+    Checks if all files in the package have valid names.
+    """
+    def get_invalid_files(path, *patterns):
```

Comment:
Are there multiple patterns, or only one?

Reply:
While I was writing this PR there was a need for multiple patterns, but now it's just one